src/shapes/object.class.js

```diff
@@ -560,6 +560,16 @@
      */
     noScaleCache:              true,
 
+    /**
+     * When `false`, the stoke width will scale with the object.
+     * When `true`, the stroke will always match the exact pixel size entered for stroke width.
+     * default to false
+     * since 2.5.1
```

Comment:
```suggestion
     * @since 2.6.0
     * @type Boolean
     * @default false
```
this is a new feature, need a minor bump.

Reply:
ops i got it wrong